roles/openshift_control_plane/tasks/main.yml

```diff
@@ -78,7 +78,7 @@
 
 - name: Create the ldap ca file if needed
   copy:
-    dest: "{{ item.ca if 'ca' in item and '/' in item.ca else '/etc/origin/master/' ~ item.ca | default('ldap_ca.crt') }}"
+    dest: "/etc/origin/master/ldap_ca.crt"
```

Comment:
Perhaps we should prefix each filename with the 'name' key of the auth_provider?  That would allow multiple auth providers of the same type?

Reply:
Good idea, implemented